make/config.mk

```diff
@@ -48,6 +48,15 @@ USE_CUDA_PATH = NONE
 # whether use CuDNN R3 library
 USE_CUDNN = 0
 
+# CUDA architecture setting: going with all of them.
+# For CUDA < 6.0, comment the *_50 lines for compatibility.
+CUDA_ARCH := -gencode arch=compute_20,code=sm_20 \
```

Comment:
I think so. based on this, we need add 6.1. https://en.wikipedia.org/wiki/CUDA
I don't have PASCAL card, so i cannot 100% sure. but I will try to make the change. if nvcc doesn't complain, it should be correct.